.github/workflows/django.yml

```diff
@@ -0,0 +1,29 @@
+name: Django CI
```

Comment:
The library has tests, it's a pity it does not have CI set up. I fixed it.

Reply:
thanks, maybe fail with travis